web/hosts_next.go

```diff
@@ -4,9 +4,25 @@ import (
 	"net/http"
 
 	"github.com/gin-gonic/gin"
+	"github.com/trento-project/trento/web/models"
 	"github.com/trento-project/trento/web/services"
 )
 
+func NewHostsNextHealthContainer(hostList models.HostList) *HealthContainer {
+	h := &HealthContainer{}
+	for _, host := range hostList {
+		switch host.Health {
```

Comment:
![image](https://user-images.githubusercontent.com/20770/141443204-42defb82-7551-4bfc-8058-539c18ed83b0.png)

Reply:
This should go away pretty soon with the new frontend, bear with me.